### DISCUSSION TITLE
reference/docs/markdown

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#741] [Q&A]
# reference/docs/markdown

https://blog.cutterman.cn/reference/docs/markdown.html

**Comment by SH-ke** [comment]
调整尺寸大小，直接在连接地址后添加形如 =600x600的数字即可调整图片大小。（推荐）

![Alt](https://link.png =600x600)

**Reply by jaywcjlove** [reply]
@SH-ke 非标准语法，在许多地方破坏图片预览
